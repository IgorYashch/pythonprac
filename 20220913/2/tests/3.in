[1765, 234, -900, 456, 12, 56, 6555, -3, 0, 1.675]
