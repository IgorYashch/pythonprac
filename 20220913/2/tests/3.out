-900, -3, 0, 1.675, 12, 56, 234, 456, 1765, 6555
